hasura.planx.uk/metadata/tables.yaml

```diff
@@ -294,7 +294,8 @@
             {
               "payload": {
                 "sessionId": {{$body.event.data.new.id}},
-                "email": {{$body.event.data.new.email}}
+                "email": {{$body.event.data.new.email}},
+                "isLocked": {{inverse(empty($body.event.data.new.locked_at))}}
```

Comment:
The lowcal submission webhook payload now includes the locked status from which we can infer if this is an invite-to-pay submission or not.